File: 742-to-lower-case/to-lower-case.cpp
class Solution {
public:
    string toLowerCase(string s) {
        string res = "";
        for (char c:s){
            if(c>='A'&&c<='Z'){
                res += c+32;
            }else {
                res+=c;
            }
        }
        return res;
    }
};

File: 742-to-lower-case/to-lower-case.java
class Solution {
    public String toLowerCase(String s) {
        String res = new String();
        for(char c:s.toCharArray()){
            if (c>='A' && c<='Z'){
                res += (char)(c+32);
            }else {
                res +=c;
            }
        }
        return res;
    }
}